Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json

```diff
@@ -40,6 +40,11 @@
                     "updated": 1638338966721
                 },
                 {
+                    "language": "TR",
+                    "text": "Bu hiyerarşiyi iki ana amaç için kullanacaksınız:",
+                    "updated": 1642504350982
+                },
+                   {
```

Comment:
The bracket before the spanish translation is not in line with the closing bracket from the Turkey one.

Reply:
fix it